crates/rpc/src/rpc.rs

```diff
@@ -146,4 +181,143 @@ impl Rpc for Server {
 
         Ok(listings.into_values().collect())
     }
+
+    fn get_stores(&self) -> Result<Vec<Storefronts>> {
+        let db = self.db()?;
+        let items: Vec<Storefront> = storefronts::table
+            .order_by(storefronts::owner_address)
+            .load(&db)
+            .map_err(internal_error("Failed to load storefronts"))?;
+        let mut stores: Vec<Storefronts> = Vec::new();
+        for Storefront {
+            owner_address,
+            subdomain,
+            title,
+            description,
+            favicon_url,
+            logo_url,
+        } in items
+        {
+            stores.push(Storefronts {
+                owner_address: owner_address.to_string(),
+                subdomain: subdomain.to_string(),
+                title: title.to_string(),
+                description: description.to_string(),
+                favicon_url: favicon_url.to_string(),
+                logo_url: logo_url.to_string(),
+            });
+        }
+        dbg!();
+        println!("{}", stores.len());
+        Ok(stores)
+    }
+
+    fn get_stores_count(&self) -> Result<i64> {
+        let db = self.db()?;
+        let count = storefronts::table.count().get_result(&db);
+        Ok(count.unwrap())
```

Comment:
```suggestion
        storefronts::table
            .count()
            .get_result(&db)
            .map_err(internal_error("Failed to count storefronts"))
```
Don't `unwrap` or `expect` Results, this will cause the server to panic.